solvexia_sdk/usergroup/usergroup.py

```diff
@@ -0,0 +1,39 @@
+import requests
+from solvexia_sdk import api
+
+class user:
```

Comment:
Just checking if this class name is intended -> class user in file usergroup.

Reply:
Not intended. I have updated it.